### DISCUSSION TITLE
Why and Why Not

### DISCUSSION THREAD

**Discussion by billyjbryant** [OhMyGuus/BetterCrewLink#38] [General]
### Discussion

I believe the Crewlink application is a great tool and can only get better by contributions. That said, I'm curious about why the features of "BetterCrewLink" aren't being PR'd back into the original CrewLink rather than maintaining a new, separate tool that will eventually diverge as paths of development differ? 

This isn't an attack, more just asking for candid discussion.

**Comment by OhMyGuus** [comment]
Most of the features are from PR's that are on Crewlink, I made this fork because they weren't being merged for weeks and people where trying to play the game with NAT issues or on the windows store/epic version of the game so they started using this fork. And at the latest update it broke even more but Ottomated got an offer from Overwolf to create an new Crewlink for them on there platform so he stopped working on the original Crewlink.  

The other reason I stopped making new PR's because Ottomated was working in a private repository without pushing changes towards a dev branch, so every time he did an update it was bloated with changes and the PR needed to be remade again basically.

**Reply by billyjbryant** [reply]
Thanks for your candor, that answered my question and was the response I had feared was the case. I would hope that with name recognition, maybe Ottomated might release control of the original CrewLink to a dev group to continue further maintenance. 

Nevertheless, I do have a few open PRs for CrewLink & CrewLink-Server that I will bring over here.
